config/locales/mfa.en.yml

```diff
@@ -13,7 +13,7 @@ en:
       phone:
         nochange: Your account is already using this mobile number. Enter a different number.
       phone_code:
-        expired: The security code you’ve entered has expired. Security codes expire after 10 minutes. We can <a class="govuk-link" href="%{resend_link}">send a new security code</a>.
+        expired: The security code you’ve entered has expired. Security codes expire after 30 minutes. We can <a class="govuk-link" href="%{resend_link}">send a new security code</a>.
```

Comment:
NB this message will now never be shown to users.
I figure as we're reviewing this journey it makes more sense to leave this in place and finally amend / remove when we've examined the data and made any follow up changes?